packages/scandipwa/src/component/FieldGroup/FieldGroup.component.js

```diff
@@ -51,7 +50,7 @@ export class FieldGroup extends PureComponent {
     //#region LABEL/TEXT RENDER
     // Renders validation error messages under group
     renderErrorMessage(message) {
-        return <div block="Field" elem="ErrorMessage">{ message }</div>;
+        return <div block="Field" key={ message } elem="ErrorMessage">{ message }</div>;
```

Comment:
Sure if message is good to show as key?

Reply:
Nono, I was testing and forgot to delete it